app/tsconfig.json

```diff
@@ -11,13 +11,13 @@
    },
    "files": [
 	  "./src/ts/app.ts",
-    "./src/ts/upload.ts",
-    "./src/ts/play.ts",
+      "./src/ts/upload.ts",
```

Comment:
Was this whitespace change to line up these files with the rest?

(If so, they're still showing up as not lined up on Github. Maybe a spaces vs. tabs issue?)